doc/source/tutorials/bipartite_matching/assets/maxflow2.py

```diff
@@ -0,0 +1,39 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph(
+    9,
+    [(0, 4), (0, 5), (1, 4), (1, 6), (1, 7), (2, 5), (2, 7), (2, 8), (3, 6), (3, 7)],
+    directed=True
+)
+
+# Assign nodes 0-3 to one side, and the nodes 4-8 to the other side
+for i in range(4):
+    g.vs[i]["type"] = True
+for i in range(4, 9):
+    g.vs[i]["type"] = False
+
+# Add source and sink as nodes 9 and 10
+g.add_vertices(2)
+g.add_edges([(9, 0), (9, 1), (9, 2), (9, 3)]) # connect source to one side
+g.add_edges([(4, 10), (5, 10), (6, 10), (7, 10), (8, 10)]) # ... and sinks to the other
+
+flow = g.maxflow(9, 10)
+print("Maximal Matching is:", flow.value)
+
+# Manually set the position of source and sink to display nicely
+layout = g.layout_bipartite()
+layout[9] = (2, -1)
+layout[10] = (2, 2)
+
+fig, ax = plt.subplots()
+ig.plot(
+    g,
+    target=ax,
+    layout=layout,
+    vertex_size=0.4,
+    vertex_label=range(g.vcount()),
+    vertex_color=["lightblue" if i < 9 else "orange" for i in range(11)],
+    edge_width=[1.0 + flow.flow[i] for i in range(g.ecount())]
```

Comment:
What version are you using? This line does not seem to work with 0.9.8. It wants a single number, not a list.

Reply:
I'm using the latest develop branch. There are a few reasons why I'm doing this:

- Fabio and I have agreed that we will be using Matplotlib over Cairo, since Cairo is being phased out.
- Matplotlib on master is lacking a few features, such as setting each edge widths individually shown above, and the need to manually remove the spines and ticks from the output file, which would unecessarily complicate the code examples.
- Since the tutorial is not coming out immediately, it makes sense to be able to use features from develop. 

Of course, the downside is that the develop branch is susceptible to change, invalidating parts of the tutorial. If you think that there's going to be alot of changes to the matplotlib drawer or have other concerns, I'm happy to rework this to use the main version.